### PR TITLE
[Ready for approve/merge] Update model download URLs to a specific commit (pin them)

### PR DESCRIPTION
**Pull request by tpmccallum**
Signed-off-by: tpmccallum tim.mccallum@fermyon.com

### CODE REVIEW

File: content/spin/serverless-ai-tutorial.md
Comment (tpmccallum) [inline content/spin/serverless-ai-tutorial.md:1]: These URLs (that pin to certain commit hashes) all work, except for the `model.safetensors` 
I am unable to locate a version for the `model.safetensors` file.
Not sure how to fetch it, other than the way in which we currently fetch it i.e. `wget https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2/resolve/refs%2Fpr%2F21/model.safetensors`

File: content/spin/serverless-ai-tutorial.md [hunk 1]
```diff
@@ -176,8 +176,8 @@ The following section fetches a specific version of the [sentence-transformers](
 ```bash
 $ mkdir -p .spin/ai-models/all-minikm-16-v2
 $ cd .spin/ai-models/all-minikm-16-v2
-$ wget https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2/resolve/refs%2Fpr%2F21/tokenizer.json
-$ wget https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2/resolve/refs%2Fpr%2F21/model.safetensors
+$ wget https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2/resolve/7dbbc90392e2f80f3d3c277d6e90027e55de9125/tokenizer.json
+$ wget https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2/resolve/7dbbc90392e2f80f3d3c277d6e90027e55de9125/model.safetensors
```
Comment (tpmccallum) [inline content/spin/serverless-ai-tutorial.md:180]:
```suggestion
$ wget https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2/resolve/refs%2Fpr%2F21/model.safetensors
```

File: content/spin/serverless-ai-tutorial.md
Comment (radu-matei) [reply content/spin/serverless-ai-tutorial.md:1]: @tpmccallum - https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2/commit/0b6dc4ef7c29dba0d2e99a5db0c855c3102310d8

Comment (tpmccallum) [reply content/spin/serverless-ai-tutorial.md:1]: Just followed that link and arrived at the specific commit for the `model.safetensors` file download. There is a new PR Re: this comment. Apologies for not making the change as part of this PR. 
https://github.com/fermyon/developer/pull/907/files

**Pull request merged**